File: AppjedrezApi/AuthApi/Models/Usuario.cs
using System;
using System.Collections.Generic;

namespace AuthApi.Models;

public partial class Usuario
{
    public Guid Id { get; set; }

    public string NombreUsuario { get; set; }

    public string Clave { get; set; }

    public string Nombres { get; set; }

    public string Apellido { get; set; }

    public string Email { get; set; }

    public string Telefono { get; set; }

    public int? IdRolUsuario { get; set; }

    public string Token { get; set; }

    public byte[] FotoPerfil { get; set; }

    public virtual RolUsuario IdRolUsuarioNavigation { get; set; }

    public virtual ICollection<InscripcionOrganizador> InscripcionOrganizadors { get; } = new List<InscripcionOrganizador>();

    public virtual ICollection<Inscripcion> Inscripcions { get; } = new List<Inscripcion>();

    public virtual ICollection<Notificacion> Notificacions { get; } = new List<Notificacion>();

    public virtual ICollection<Partidum> PartidumJugadorBlancasNavigations { get; } = new List<Partidum>();

    public virtual ICollection<Partidum> PartidumJugadorNegrasNavigations { get; } = new List<Partidum>();

    public virtual ICollection<Torneo> Torneos { get; } = new List<Torneo>();
}

File: appjedrez/src/app/models/usuario.ts
import { Inscripcion } from './inscripcion';
import { Partida } from './partida';
import { RolUsuario } from './rol-usuario';
import { Torneo } from './torneo';

export interface Usuario {
  id: string;
  nombreUsuario: string;
  clave: string;
  nombres: string;
  apellido: string;
  email: string;
  telefono: string;
  idRolUsuario: number;
  token: string;
  elo: number;
  rolUsuario: RolUsuario;
  inscripciones: Inscripcion[];
  notificaciones: Notification[];
  partidasJugadorBlancas: Partida[];
  partidasJugadorNegras: Partida[];
  torneos: Torneo[];
}
